packages/expo-ui/components/Progress/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { requireNativeView } from 'expo';
+import { StyleProp, ViewStyle } from 'react-native';
+
+export type ProgressProps = {
+  /**
+   * Custom styles for the progress component.
+   */
+  style?: StyleProp<ViewStyle>;
+  /**
+   * The current progress value of the slider. This is a number between 0 and 1.
+   */
+  progress?: number | null;
+  /**
+   * Progress color.
+   */
+  color?: string;
+
+  /**
+   * Track color.
+   *
+   * @platform android
+   */
+  trackColor?: string;
```

Comment:
I would suggest wrapping that in a record `elementColors` for consistency with other components – wdyt?